packages/app-security/src/components/NotAuthorizedError.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from "react";
+import { Link } from "@webiny/react-router";
+import { css } from "emotion";
+import styled from "@emotion/styled";
+import Helmet from "react-helmet";
+import authErrorImg from "../admin/assets/images/SecureRouteError.svg";
+import { Typography } from "@webiny/ui/Typography";
+
+const ContentWrapper = styled("div")({
+    display: "block",
+    paddingTop: "15%",
+    textAlign: "center",
+    margin: "auto"
+});
+
+const styles = {
+    authErrorImgStyle: css({
+        width: "192px"
+    }),
+    bodyStyle: css({
+        color: "var(--mdc-theme-text-primary-on-background)"
+    }),
+    linkStyle: css({
+        textDecoration: "none",
+        "&:hover": {
+            textDecoration: "none"
+        }
+    }),
+    divStyles: css({
+        padding: "5px"
+    })
+};
+
+const NotAuthorizedError = () => {
+    return (
+        <ContentWrapper>
+            <Helmet title={"Not authorized"} />
+            <div className={styles.divStyles}>
+                <img className={styles.authErrorImgStyle} src={authErrorImg} alt="Not Authorized" />
+            </div>
+
+            <div className={styles.divStyles}>
```

Comment:
I think there is no need to wrap each tag/element in a `div`. What do you think? :slightly_smiling_face:

Reply:
I actually added that coz the `<Typography>` package renders `<span>` instead of `<div>`. And if we are using <div> it all comes on the same line. Is there any way to fix that?

Reply:
![Screenshot from 2020-10-07 20-43-49](https://user-images.githubusercontent.com/45600289/95350517-de947d00-08dd-11eb-8c98-8dd2cf27630d.png)
Please look into this.

Reply:
There are multiple ways to fix that but I'll go with the simplest i.e apply `display: block` CSS property on the `Typography` component. :slightly_smiling_face: